packages/dialect-react-ui/components/common/primitives.tsx

```diff
@@ -214,11 +214,16 @@ export function Toggle({
 }
 
 export const useBalance = () => {
```

Comment:
Wdyt of moving this to the react-sdk as well?

Reply:
hm, not sure yet, but we'll think, good point!